src/load_inputs/load_resources_data.jl

```diff
@@ -1109,33 +1125,36 @@ function summary(rs::Vector{<:AbstractResource})
 end
 
 """
-    load_resources_data!(inputs::Dict, setup::Dict, case_path::AbstractString)
+    load_resources_data!(inputs::Dict, setup::Dict, case_path::AbstractString, resources_path::AbstractString)
 
-This function loads resources data from the `setup["ResourcePath"]` folder and create the GenX data structures and add them to the `inputs` `Dict`. 
+This function loads resources data from the resources_path folder and create the GenX data structures and add them to the `inputs` `Dict`. 
 
 # Arguments
 - `inputs (Dict)`: A dictionary to store the input data.
 - `setup (Dict)`: A dictionary containing GenX settings.
-- `case_path (AbstractString)`: The path to the case.
+- `case_path (AbstractString)`: The path to the case folder.
+- `resources_path (AbstractString)`: The path to the case resources folder.
 
 Raises:
     DeprecationWarning: If the `Generators_data.csv` file is found, a deprecation warning is issued, together with an error message.
 """
-function load_resources_data!(inputs::Dict, setup::Dict, case_path::AbstractString)
+function load_resources_data!(inputs::Dict, setup::Dict, case_path::AbstractString, resources_path::AbstractString)
     if isfile(joinpath(case_path, "Generators_data.csv"))
         msg = "The `Generators_data.csv` file was deprecated in release v0.4. " *
             "Please use the new interface for generators creation, and see the documentation for additional details."
         Base.depwarn(msg, :load_resources_data!, force=true)
         error("Exiting GenX...")
     else
         # create vector of resources from dataframes
```

Comment:
Since you throw an `error` in the `if` clause, could the contents of `else` be moved out of the `else` and up one indent level? After all, they're on the Happy Path.